daemon/algod/api/algod.oas2.json

```
@@ -3779,6 +3843,9 @@
         },
         "exec-trace": {
           "$ref": "#/definitions/SimulationTransactionExecTrace"
+        },
+        "unnamed-resources": {
```

Again, maybe.  Seems to fit the idea that here we are conveying what actually happened.
```suggestion
        "unnamed-resources-accessed": {
```

---

same, f343aa459